### DISCUSSION TITLE
How to restrict redash users from logging in to redash after they enter the wrong password for three times?

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6095] [Q&A]
To improve security, want to have user account locked after password failure for three times.

**Comment by Jengro777** [comment]
It is already difficult to merge the code here. You have another option available:
https://github.com/RedashCommunity/redash

**Reply by qq20230530** [reply]
Thank you.
Do you mean that I would be able to set password failure restrictions if I use the version you sent to me?
Could you please tell me more details?
